### PR TITLE
EnvGen: ENVGEN_NOT_STARTED check moved after check for mDone

### PR DESCRIPTION
**Pull request by sonoro1234**
## Purpose and Motivation

Fixing https://github.com/supercollider/supercollider/issues/4781

### CODE REVIEW

**Comment by patrickdupuis** [comment]
This fixes #4781 for me, but I can't offer a review of the proposed changes.

File: server/plugins/LFUGens.cpp [hunk 1]
```diff
@@ -2422,10 +2422,7 @@ static inline bool check_gate_ar(EnvGen* unit, int i, float& prevGate, float*& g
 }
 
 static inline bool EnvGen_nextSegment(EnvGen* unit, int& counter, double& level) {
-    if (unit->m_stage == ENVGEN_NOT_STARTED) {
-        counter = INT_MAX;
-        return true;
-    }
+
```
Comment (brianlheim) [inline server/plugins/LFUGens.cpp]: you need to remove this line

**Comment by sonoro1234** [comment]
Redone as
https://github.com/supercollider/supercollider/pull/4789

**Comment by salkin-mada** [comment]
nice! thank you so much

**Pull request closed without merging**